diem-move/diem-framework/core/transactional-tests/block/block_prologue.move

```diff
@@ -0,0 +1,36 @@
+//# init --validators Vivian
+
+//# block --proposer Vivian --time 1000000
+
+//# run --admin-script
+//#     --signers DiemRoot
+script{
+use DiemFramework::DiemTimestamp;
+use DiemFramework::DiemBlock;
+
+fun main() {
+    assert!(DiemBlock::get_current_block_height() == 1, 73);
+    assert!(DiemTimestamp::now_microseconds() == 1000000, 76);
+}
+}
+
+//# run --admin-script
+//#     --signers DiemRoot
+script{
+use DiemFramework::DiemTimestamp;
+
+fun main() {
+    assert!(DiemTimestamp::now_microseconds() != 2000000, 77);
+}
+}
+
+// REVIEW: what is this doing? Shall we delete it?
```

Comment:
What is the 20 there? Is it resetting the time to 20? Does this interact with the block prologue in some way?

Reply:
The block prologue sets the timestamp, and this seem to do the same thing. Although I really don't think we should do this via an admin script, hence I'm asking if we should delete or rewrite it.